src/Pages/Rockets.js

```diff
@@ -0,0 +1,62 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import {
+  fetchRockets,
+  reserveRocket,
+  cancelReserve,
+} from '../redux/rockets/rocketsSlice';
+
+const Rockets = () => {
+  const dispatch = useDispatch();
+  const rockets = useSelector((state) => state.rockets);
+
+  const handleReserveRocket = (rocketId, rocketName) => {
+    dispatch(reserveRocket(rocketId, rocketName));
+  };
+
+  const handleCancelReserve = (rocketId) => {
+    dispatch(cancelReserve(rocketId));
+  };
+
+  useEffect(() => {
+    dispatch(fetchRockets());
+  }, [dispatch]);
+
+  return (
+    <section className="rockets-container">
+      {rockets.map((rocket) => (
+        <li className="rocket" key={rocket.id}>
+          <img className="rocket-img" src={rocket.flickr_images} alt={rocket.name} />
+          <article className="rocket-info">
+            <h2 className="rocket-name">{rocket.name}</h2>
+            <p className="rocket-des">
+              {rocket.reserved ? (
```

Comment:
While you did an impressive job working on the project requirements it would be nice to follow design requirements please follow the design framework for this project I understand you are trying to be creative but it is also important to follow instructions. Please add the appropriate logo and name for this project and follow the design as it is. 

This is your design:
![Screenshot 2023-04-14 at 8 03 39 PM](https://user-images.githubusercontent.com/79506105/232124182-e981abb0-e59e-4345-810f-dde3f077d2c1.png)


Please kindly stick to this design:




![Home page ](https://user-images.githubusercontent.com/79506105/232124287-5b1a4d08-3401-4efb-a470-ef8f86a8ef60.png)